include/nuttx/efuse/esp_efuse.h

```diff
@@ -0,0 +1,184 @@
+/****************************************************************************
+ * include/nuttx/efuse/esp_efuse.h
```

Comment:
Can we define a generic efuse driver interface? So not only ESP but also other chip can implement this interface.